server/routes/termdb.config.ts

```diff
@@ -68,17 +70,14 @@ returns:
 	a json object
 */
 
-function make(q, res, ds: Mds3WithCohort, genome) {
+function make(q, req, res, ds: Mds3WithCohort, genome) {
 	const tdb = ds.cohort.termdb
 	// add attributes to this object to reveal to client
 
-	// TODO: may fill this in later with other request-specific credentials
-	const auth = { embedder: q.embedder }
-
 	// add required attributes
 	const c: any = {
 		selectCohort: tdb.selectCohort, // optional
-		supportedChartTypes: tdb.q?.getSupportedChartTypes(auth),
+		supportedChartTypes: tdb.q?.getSupportedChartTypes(getClientAuth(q, req)),
```

Comment:
If `forbiddenRoutes` is returned together with `clientAuthResults` by a new authApi method, then should remain as one argument.